docs/experiments-ltr-msmarco-document-reranking.md

```diff
@@ -43,11 +43,13 @@ python -m pyserini.search.lucene.ltr \
   --max-passage --hits 10000
 ```
 
+Note that internally, retrieval depends on tokenization with spaCy; our implementation currently depends on v3.2.1 (this is potentially important as tokenization might change from version to version).
+
 After the run finishes, we can evaluate the results using the official MS MARCO evaluation script:
 
 ```bash
-$ python tools/scripts/msmarco/msmarco_doc_eval.py \
-    --judgments tools/topics-and-qrels/qrels.msmarco-doc.dev.txt \
+$ python -m pyserini.eval.msmarco_doc_eval \
+    --judgments msmarco-doc-dev \
     --run runs/run.ltr.msmarco-doc.tsv
 
 #####################
```

Comment:
There is a small MRR@10 change, here is what I got:
#####################
MRR @100: 0.31088166784756843
QueriesRanked: 5193
#####################